apidef/oas/default.go

```diff
@@ -226,9 +228,10 @@ func GetTykExtensionConfigParams(r *http.Request) *TykExtensionConfigParams {
 	authentication := getQueryValPtr(strings.TrimSpace(queries.Get("authentication")))
 	validateRequest := getQueryValPtr(strings.TrimSpace(queries.Get("validateRequest")))
 	allowList := getQueryValPtr(strings.TrimSpace(queries.Get("allowList")))
+	mockResponse := getQueryValPtr(strings.TrimSpace(queries.Get("mockResponse")))
 
 	if upstreamURL == "" && listenPath == "" && customDomain == "" && apiID == "" &&
```

Comment:
There has to be a nicer check we can do. Why not always allocate the config params return? As the queryValPtrs are passed along as `*bool`, do we need to check them for nil?